frontend/appflowy_flutter/lib/startup/tasks/windows.dart

```diff
@@ -18,6 +19,7 @@ class InitAppWindowTask extends LaunchTask with WindowListener {
 
   @override
   Future<void> initialize(LaunchContext context) async {
+    final SharedPreferences prefs = await SharedPreferences.getInstance();
```

Comment:
Please don't use `SharedPreferences` directly.

There's a already existing window manager. Use it instead.

Reply:
I use it because I create a variable using it that goes true if the window is maximized...
What do you mean using the window manager?

Reply:
I pushed the code, please take a look.